### DISCUSSION TITLE
Troponin essay?

### DISCUSSION THREAD

**Discussion by santiagoferriere** [MIT-LCP/mimic-code#1673] [MIMIC-IV]
I need info. in the characteristics of Troponin essays used in  MIMIC III and IV, (are they high-sensitivity, whats their Limit of detection?), Any idea how i could obtain this information?

**Comment by Sophiejohny** [comment]
MIMIC III and IV, (are they high-sensitivity, whats their Limit of detection?), Any idea  [https://www.dumpscafe.com/Braindumps-N10-009.html](https://www.dumpscafe.com/Braindumps-N10-009.html)
